### PR TITLE
Add weasyprint to python.yaml

### PR DESCRIPTION
**Pull request by bullerbeck**
Please add the following dependency to the rosdep database.

## Package name:

weasyprint

## Package Upstream Source:

https://pypi.org/project/weasyprint/

## Purpose of using this:

This dependency is a visual rendering engine for HTML and CSS that can export to PDF. It is useful for generating reports out of databases filled with e.g. sensor data from robots. This is why I think it's valuable to be added to the rosdep database. 

Distro packaging links:

https://files.pythonhosted.org/packages/d4/ca/b5c774f85ef78469e729927ba64af3110a005d269fbfaf90d00bfd80cf64/weasyprint-53.1-py3-none-any.whl

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here: 

http://sourcecode_url

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

**Review by audrow — APPROVED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5494,6 +5494,13 @@ python-watchdog:
     xenial: [python-watchdog]
     yakkety: [python-watchdog]
     zesty: [python-watchdog]
+python-weasyprint-pip:
+  debian:
+    pip:
+      packages [weasyprint]
```
Comment (ivanpauno) [inline rosdep/python.yaml:5500]:
```suggestion
      packages: [weasyprint]
```

File: rosdep/python.yaml [hunk 2]
```diff
@@ -5494,6 +5494,13 @@ python-watchdog:
...
+  ubuntu:
+    pip:
+      packages [weasyprint]
```
Comment (ivanpauno) [inline rosdep/python.yaml:5503]:
```suggestion
      packages: [weasyprint]
```

File: rosdep/python.yaml [hunk 3]
```diff
@@ -5494,6 +5494,13 @@ python-watchdog:
     xenial: [python-watchdog]
     yakkety: [python-watchdog]
     zesty: [python-watchdog]
+python-weasyprint-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:5497]: do you want to add an entry for python3 or python2?
For python3 please use `python3-weasyprint-pip`.

The package seems to only be available for python3.

**Comment by hidmic** [comment]
@bullerbeck friendly ping

File: rosdep/python.yaml (hunk 3 above)
Comment (bullerbeck) [reply rosdep/python.yaml]: changed

**Review by hidmic — APPROVED** [review]

**Review by ivanpauno — APPROVED** [review]

**Pull request merged**